packages/vkui/src/components/Pagination/PaginationNavigationButton/PaginationNavigationButton.tsx

```diff
@@ -1,57 +1,77 @@
 import * as React from 'react';
-import { Button } from '../../Button/Button';
+import { Button, ButtonProps } from '../../Button/Button';
 import { VisuallyHidden } from '../../VisuallyHidden/VisuallyHidden';
 
-export interface PaginationNavigationButtonProps {
-  type: 'prev' | 'next';
-  style: 'icon' | 'caption' | 'both';
-  caption: React.ReactNode;
-  Icon: React.ComponentType;
-  a11yLabel: React.ReactNode;
-  disabled?: boolean;
-  onClick: () => void;
+export interface PaginationNavigationButtonOpts {
+  'type': 'prev' | 'next';
+  'style': 'icon' | 'caption' | 'both';
+  'caption': React.ReactNode;
+  'Icon': React.ComponentType;
+  'a11yLabel': React.ReactNode;
+  'disabled'?: boolean;
+  'onClick': (event: React.MouseEvent<HTMLElement>) => void;
+  'data-page': number | undefined;
```

Comment:
```suggestion
  'data-page': number;
```
давай лучше сделаем обязательным – мы же его всегда передаём

Reply:
для кнопки prev data-page будет undefined, если сейчас активна активная страница. аналогично для next